decode.go

```diff
@@ -253,3 +253,8 @@ func NewDecoder(r io.ReadCloser) (*Decoder, error) {
 func Decode(r io.ReadCloser) (*Decoder, error) {
 	return NewDecoder(r)
 }
+
+// Decode decodes all mp3 data from d into dst - memory usage can be large, use with caution
+func (d *Decoder) Decode(dst io.Writer) (int64, error) {
+	return io.Copy(dst, d)
```

Comment:
I wouldn't say we _need_ it, I just think it's a nice shortcut to have. I suppose letting the user use io.Copy isn't much different, I just like the idea of reducing the number of parameters, and personally I always confuse (dst, src) as (src, dst) because other languages use src first.